src/app.ts

```diff
@@ -47,13 +47,16 @@ export class App implements Container {
   }
 
   addOutageMonitor(name: string, { tags, message, ...monitor }: Monitor) {
+    const messageWithAlertContactInserted = message.replace(
+      /\{\{\/is_(?!warning)/g,
+      ` ${this.team.alertContact}{{/is_`,
+    );
+
```

Comment:
🤔  I wonder if this idea should be expanded to each individual contact; e.g. `team.contact = {alert: "", warning: "", ..etc. }`; maybe with a `default` or `all` contact that is used first, overridden by second declaration, otherwise left blank

```
app = new App({
    team: {
        defaultContact: "@testdefaultContact",
        warningContact: "",
        alertContact: "@testAlertContact",
    },
});
// =>
message:
    "{{#is_alert}}alert message @testAlertContact{{/is_alert}}" +
    "{{#is_warning}}warning message{{/is_warning}}" +
    "{{#is_recovery}}recovery message @testdefaultContact{{/is_recovery}}" +
    " @testdefaultContact",
```

Reply:
It's a good point @Sonna but that would likely require all the different monitor types to be refactored so everything is consistent, could be a future improvement for the owning team to consider I'd say but should be out of scope for now I think.